src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,7 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    private static void main(String[] args) {
+
```

Comment:
Don't see this step: `Remove this comment and test your Builder implementation here...`